src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -656,6 +660,22 @@ def apply_snmp_gateway(self, spec: SNMPGatewaySpec) -> OrchResult[str]:
         """Update an existing snmp gateway service"""
         raise NotImplementedError()
 
+    def apply_elasticsearch(self, spec: TracingSpec) -> OrchResult[str]:
+        """Update an existing ElasticSearch daemon(s)"""
+        raise NotImplementedError()
+
+    def apply_jaeger_agent(self, spec: ServiceSpec) -> OrchResult[str]:
+        """Update an existing jaeger_agent daemon(s)"""
+        raise NotImplementedError()
+
+    def apply_jaeger_collector(self, spec: ServiceSpec) -> OrchResult[str]:
+        """Update an existing jaeger_collector daemon(s)"""
+        raise NotImplementedError()
+
+    def apply_jaeger_query(self, spec: ServiceSpec) -> OrchResult[str]:
+        """Update an existing jaeger_query daemon(s)"""
+        raise NotImplementedError()
+
```

Comment:
This is getting out of hand. I think we have to drop those completely.

Reply:
done [c8eb218](https://github.com/ceph/ceph/pull/44493/commits/c8eb218efff5f003a3b7842d4b93adc980a1c042)